src/index.js

```diff
@@ -68,7 +68,7 @@ function addTask(description) {
   const newTask = {
     description,
     completed: false,
-    index: newIndex
+    index: newIndex,
```

Comment:
Please review the code related to the newTask object I can the repetition there, can you please correct it if its a typo or you can walk me through this part of the code if its not typo

Reply:
Thanks a lot for your insight I will definitely look into now Thanks for letting me know